internal/server/turn.go

```diff
@@ -102,12 +102,19 @@ func (s *SOMASServer) endOfTurn() error {
 		return errors.Errorf("Failed to run orgs end of turn: %v", err)
 	}
 
+	err = s.runDummyHunt()
```

Comment:
Will this eventually be exchanged for a serverFunction callling the clients for hunting decisions?

Reply:
Indeed!